Doxyfile

```diff
@@ -810,7 +798,7 @@ WARN_NO_PARAMDOC       = NO
 # a warning is encountered.
 # The default value is: NO.
 
-WARN_AS_ERROR          = NO
+WARN_AS_ERROR          = YES
```

Comment:
I know the userstory says to use this flag, but the output produced when this flag is set is not very informative. When a doxygen run fails, I would expect to see all the failures my commit produced, not just the first one generated.